src/operator/fill.rs

```diff
@@ -0,0 +1,99 @@
+use crate::array::Array;
+use crate::hardware::Hardware;
+use crate::operator::Operator;
+use crate::result::Result;
+use crate::shape::Shape;
+use std::cell::RefCell;
+
+/// Fill operator: creates an array with specific hardware/shape, filled by a single value.
+pub(crate) struct Fill<'hw> {
+    /// Hardware to host the value.
+    hardware: &'hw RefCell<dyn Hardware>,
+
+    /// Shape of the resulting array.
+    shape: Shape,
+
+    /// Value of elements in the resulting array.
+    value: f32,
+}
+
+impl<'hw> Fill<'hw> {
+    pub(crate) fn new(hardware: &'hw RefCell<dyn Hardware>, shape: Shape, value: f32) -> Self {
+        Self {
+            hardware,
+            shape,
+            value,
+        }
+    }
+}
+
+impl<'hw> Operator<'hw> for Fill<'hw> {
+    fn name(&self) -> String {
+        "Fill".to_string()
```

Comment:
nit:  Shouldn't we use `String::from` for consistency? Existing other operators use that method.

```suggestion
        String::from("Fill")
```